webui/react/src/pages/Cluster/ClusterOverallStats.tsx

```diff
@@ -59,21 +78,36 @@ export const ClusterOverallStats: React.FC = () => {
             {auxContainers.running} <small>/ {auxContainers.total}</small>
           </OverviewStats>
         ) : null}
-        {activeExperiments ? (
-          <OverviewStats title="Active Experiments">{activeExperiments}</OverviewStats>
-        ) : null}
-        {activeTasks.notebooks ? (
-          <OverviewStats title="Active JupyterLabs">{activeTasks.notebooks}</OverviewStats>
-        ) : null}
-        {activeTasks.tensorboards ? (
-          <OverviewStats title="Active TensorBoards">{activeTasks.tensorboards}</OverviewStats>
-        ) : null}
-        {activeTasks.shells ? (
-          <OverviewStats title="Active Shells">{activeTasks.shells}</OverviewStats>
-        ) : null}
-        {activeTasks.commands ? (
-          <OverviewStats title="Active Commands">{activeTasks.commands}</OverviewStats>
-        ) : null}
+        <OverviewStats title="Active Experiments">
+          {Loadable.match(activeExperiments, {
+            Loaded: (activeExperiments) => activeExperiments.pagination?.total ?? 0,
+            NotLoaded: (): ReactNode => <Spinner />,
+          })}
+        </OverviewStats>
```

Comment:
so it looks like these changes result in them being displayed with a spinner instead of being conditionally rendered? i like it better.

Reply:
✅